modules/system/lang/cs/client.php

```diff
@@ -65,13 +63,24 @@
         'group' => [
             'all' => 'Vše'
         ],
+        'scopes' => [
+            'apply_button_text' => 'Filtrovat',
+            'clear_button_text'  => 'Zrušit'
```

Comment:
Please ensure that the last item in an array has a comma too. This isn't enforced anywhere right now but it will trigger the checks to run again and I prefer it to be that format anyways.